src/tokenizer.scss

```diff
@@ -38,7 +44,7 @@ $block: #{$fd-namespace}-tokenizer;
   &__input {
     border: none !important;
     min-width: 3rem !important;
-    padding: 0 $fd-tokenizer-spacing 0 0 !important;
+    padding: 0 !important;
```

Comment:
Did we want to keep with the `important!` pattern here or switch to the new pattern we did with shellbar buttons?

Reply:
Now it's handled like on shellbar.